xla/service/gpu/tests/gpu_hlo_runner_test.cc

```diff
@@ -41,7 +41,9 @@ class HloRunnerTest : public GpuCodegenTest {};
 TEST_F(HloRunnerTest, RunSingle) {
 
   std::ifstream ifs("input.hlo");
-  ASSERT_TRUE(ifs.good());
+  if(ifs.fail()) {
+    GTEST_SKIP() << "No input HLO file provided!";
+  }
```

Comment:
This test actually is used for us to evaluate hlo and backporting, which is not supposed to be build/test. Maybe we can add `["no_oss"]` tag for this file in [BUILD](https://github.com/ROCm/xla/blob/rocm-jaxlib-v0.4.31-qa-misc-backport/xla/service/gpu/tests/BUILD#L226) file